src/Workspaces/Core/Portable/Workspace/Solution/Checksum.cs

```diff
@@ -136,8 +149,8 @@ public static string GetChecksumsLogInfo(IEnumerable<Checksum> checksums)
         /// This structure stores the 20-byte SHA 1 hash as an inline value rather than requiring the use of
```

Comment:
```suggestion
         /// This structure stores the 20-byte SHA256 hash as an inline value rather than requiring the use of
```

Reply:
SHA256 doesn't produce 20 byte values; I suspect we should revise the comment that we're storing the truncated portion.

Reply:
+1

---
In reply to: [256607563](https://github.com/dotnet/roslyn/pull/33363#discussion_r256607563) [](ancestors = 256607563)